Build/GetWix.ps1

```diff
@@ -0,0 +1,21 @@
+function GetWix {

```

Comment:
I'll simplify the Invoke-WebRequest call to use OutFile, but AFAIK using Shell.Application is the easiest/most portable way to extract the contents of a ZIP file. (Last time I did this I wrote it in Python, but that's an unnecessary dependency for this project.)